src/RestClient/RestClient/src/Requests/BaseRestClientRequest.cs

```diff
@@ -151,15 +151,15 @@ protected async Task<Error> GetErrorAsync(HttpResponseMessage message)
             if (message.IsSuccessStatusCode)
                 return null;
 
-            if (message.Content != null)
-            {
-                var content = await message.Content.ReadAsStringAsync().ConfigureAwait(false);
+            var contentString = await message.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
Previously `ReadAsStringAsync` wouldn't be called on 'message.Content` if it was null, will this be fine?

Reply:
yeah i think so. `ParseResponseAsync` doesn't check Content either